Wikibase/ruleset.xml

```diff
@@ -4,9 +4,21 @@
 		https://github.com/wikimedia/mediawiki-tools-codesniffer/blob/master/MediaWiki/ruleset.xml
 		-->
 	<rule ref="vendor/mediawiki/mediawiki-codesniffer/MediaWiki">
+		<!-- This sniff is way to rigorous about way to many details that need exceptions, e.g.
+			obvious function headers that don't need a PHPDoc block. -->
+		<exclude name="MediaWiki.Commenting.FunctionComment" />
+
+		<!-- We disagree with the idea of certain characters making comments "illegal" and blocking
+			patches from being merged. This behaves especially bad on commented out code. -->
+		<exclude name="MediaWiki.Commenting.IllegalSingleLineComment" />
```

Comment:
As far as I understood this is not about commented code at all. What I found in tests is this:
```
	/*** This should fail */
	/*** This should fail */
	/* This also should fail **/
```
I don't see how is it harmful in any way.

Speaking about commented out code - I am strongly against merging commented out code to master, so if there is a sniff finding this I think it should definitely be enabled.